core/whistle-1.0.0/src/api/wapi_authn.erl

```diff
@@ -229,12 +236,14 @@ get_auth_user(ApiJObj) ->
 -spec get_auth_realm(wh_json:object() | wh_proplist()) -> ne_binary().
 get_auth_realm(ApiProp) when is_list(ApiProp) ->
```

Comment:
Note that this code does not get called anymore. I kept (and standardized it) as it is pretty similar to the above `get_auth_user`